core/src/main/scala/org/apache/spark/scheduler/Schedulable.scala

```diff
@@ -39,6 +39,7 @@ private[spark] trait Schedulable {
   def stageId: Int
   def name: String
 
+  def updateAvailableSlots(numSlots: Float): Unit
```

Comment:
Looks like slots are equivalent to cores below? Shall we just use cores?

Reply:
slots is equivalent to cores/(cores per task)